crates/router/src/connector/adyen/transformers.rs

```diff
@@ -92,25 +92,36 @@ pub enum AuthType {
 #[derive(Clone, Default, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct AdditionalData {
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
@cookieg13 we can make use `skip_serializing_none` here! ref - https://docs.rs/serde_with/latest/serde_with/attr.skip_serializing_none.html